tests/serial/map-if-necessary/two-solvers/without-substeps/ParallelExplicit.cpp

```diff
@@ -0,0 +1,33 @@
+#ifndef PRECICE_NO_MPI
+
+#include "../helper.hpp"
+#include "testing/Testing.hpp"
+
+#include <precice/precice.hpp>
+#include <vector>
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(Serial)
+BOOST_AUTO_TEST_SUITE(MapIfNecessary)
+BOOST_AUTO_TEST_SUITE(TwoSolvers)
+BOOST_AUTO_TEST_SUITE(WithoutSubsteps)
+
+BOOST_AUTO_TEST_CASE(ParallelExplicit)
+{
+  PRECICE_TEST("One"_on(1_rank), "Two"_on(1_rank));
+
+  // We receive the new start sample
+  std::vector<int> readMappings{1, 1, 1};
+  // We write the sample from the end of the time window
+  std::vector<int> writeMappings{1, 1, 1};
```

Comment:
Similar here, the last write mapping, is never exchanged to the other participant, is it?

Reply:
The last time window is still exchanged as usual. This allows to read the data after `isCouplingOngoing() == false`.